netket/graph/grid.py

```diff
@@ -43,6 +43,58 @@ def __hash__(self):
         return hash((self.shifts, self.dims))
 
 
+@dataclass
+class Planar_Rotation(Element):
+    def __init__(self, info, dims):
+        self.num_quarter_rots, self.axes = info
+        self.dims = dims
+
```

Comment:
Since this is a [`@dataclass`](https://docs.python.org/3/library/dataclasses.html), it probably should not have a custom `__init__`. (I was mainly using dataclasses here because they give a useful constructor and equality comparison by default.) Instead, the attributes should be specified like this (the code calling the constructor needs to be modified as well since this replaces `info` with two separate parameters):
```suggestion
    num_quarter_rots: int
    axes: Tuple[int]
    dims: Tuple[int]
```